CMS/templates/base.html

```diff
@@ -20,6 +20,7 @@
     <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
     <link href="{% sass_src 'scss/global.scss' %}" rel="stylesheet" type="text/css" >
+    <link href="{% static 'css/crc.css' %}" rel="stylesheet" type="text/css" >
```

Comment:
Have you moved the file from scss to css?
The css files are git ignored, so the styling is not in the repo

Reply:
yes I have - something is going on with that css and it won't compile as scss

Reply:
have updated that with an ignore line in the .gitignore

suggest we move to scss as soon as it becomes an option

Reply:
there are a couple of what seem to be typos in the css that scss doesn't like but the css doesn't care. This is fine for now.